src/index.html

```diff
@@ -58,61 +60,120 @@ <h2>
+          <ul class="my-bookings"></ul>
+          <button type="button" class="new-booking-btn">New Booking</button>
+        </div>
+        <button type="button" class="logout">Logout</button>
+      </section>
+    </section>
+
+    <section class="booking-page hide">
       <div class="dashboard-section">
         <h3>New Booking</h3>
-        <form
-          action="https://fe-apps.herokuapp.com/api/v1/overlook/1904/bookings/bookings"
-          method="post"
-        >
+        <form action="" method="GET" class="date-form">
           <label for="select-date">Select date to view available rooms:</label>
           <input
             type="date"
             id="select-date"
+            class="date-field"
             name="trip-date"
             value="yyyy-MM-dd"
             min="2020-01-01"
             max="2022-12-31"
           />
-        </form>
-      </div>
-      <button type="button" class="logout">Logout</button>
-    </section>
-
-    <section class="guest-dashboard hide">
-      <div class="dashboard-welcome">
-        <h2>
-          Welcome, <span class="guest-name"></span>! Today is
-          <span class="today-date"></span>.
-        </h2>
-      </div>
+          <button type="button" class="date-submit">Search Rooms</button>
 
-      <div class="dashboard-section">
-        <h3>Spending</h3>
-        <p class="spending"></p>
-      </div>
+          <label for="room-filter">Choose a room type:</label>
+          <select name="room-type" id="room-filter">
+            <option value="">-Please choose room type-</option>
+            <option value="single room">Single Room</option>
+            <option value="junior suite">Junior Suite</option>
+            <option value="suite">Suite</option>
+            <option value="residential suite">Residential Suite</option>
+          </select>
+          <button type="button" class="filter-submit">Filter Rooms</button>
+        </form>
+        <ul class="available-rooms"></ul>
 
-      <div class="dashboard-section">
-        <h3>My Bookings</h3>
-        <ul class="my-bookings">
-        </ul>
+        <div class="booking-form">
+          <form
+            action="https://fe-apps.herokuapp.com/api/v1/overlook/1904/bookings/bookings"
+            method="POST"
+            class="booking-form"
+          >
+            <label for="ID-input">Enter Guest ID</label>
+            <input
+              type="number"
+              name="guestID"
+              id="ID-input"
+              class="form-field"
+              placeholder="Guest ID"
+              required
+            />
+            <label for="date-input">Confirm Date</label>
+            <input
+              type="date"
+              name="guestID"
+              id="date-input"
+              class="form-field"
+              value="yyyy-MM-dd"
+              min="2020-01-01"
+              max="2022-12-31"
+              required
+            />
+            <label for="room-input">Room Number</label>
```

Comment:
Are you going to have the user manually input the room number?

Reply:
As of now, yes, future iteration they could also click a button to book specific room